utils/math.hh

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+/*
+ * Copyright (C) 2023-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include <cmath>
+
+namespace utils {
+
+template <unsigned i>
+constexpr double const_log() {
+    return std::log(i);
+}
+
```

Comment:
I don't understand why we're adding such trivial helpers.

Reply:
dropped in v2